.storybook/theo-loader.js

```diff
@@ -0,0 +1,36 @@
+const { resolve } = require('path');
+const theo = require('theo');
+const yaml = require('js-yaml');
+
+function loader(source) {
```

Comment:
It'd be nice to add a JSDoc block here, like you did in `theo-importer.js`

Reply:
Good idea!